README.md

```diff
@@ -1 +1,15 @@
-# DSCI_522_Group21
\ No newline at end of file
+# Fatality of Canadian Motor Vehicle Collisions Predictor
+
+## Project Proposal
+
```

Comment:
A couple of general comments:
1. Add titles to make the proposal more presentable (Introduction, Data, Research, EDA, Results)
2. Elaborate more on the content - seems too short. I will send across some content as well.

Reply:
1. Are you sure we should be separating everything? I was under the impression that the project proposal was a little thing as if you look at the example project the about (which contains most of the information that would be found in the proposal) is very short and is only one section
2. Not sure how long this is actually supposed to be but yeah if you send me stuff I will add it

Reply:
Actually yah looking at the example project again, I think our project proposal is an appropriate length. Tiffany's about section is a combination of the project proposal and the results they ended up getting, and it is still shorter than our project proposal

Reply:
Lets elaborate as per the Introduction section of example milestone 1: https://github.com/ttimbers/breast_cancer_predictor/tree/v1.1